multi-node/aws/pkg/config/config.go

```diff
@@ -413,10 +504,18 @@ func getContextString(buf []byte, offset, lineCount int) string {
 	return string(buf[leftLimit:rightLimit])
 }
 
+type etcdInstance struct {
+	IPAddress   net.IP
+	SubnetIndex int
```

Comment:
Yeah that would be a more robust way of going about it- I'll change that. The re-ordering of subnets thing would be kind of a stupid constraint.